### PR TITLE
Seeding

### PR DESCRIPTION
**Pull request by VinnieGlaser**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const mongoose = require('mongoose');
+const mongoUri = 'mongodb://localhost/airtreentree';
+
+const db = mongoose.connect(mongoUri);
+
+module.exports = db;
```
Comment (jscholar) [inline database/index.js:6]: I like this, separating out the connection into it's own module/file.

File: package.json [hunk 2]
```diff
@@ -4,7 +4,11 @@
   "description": "",
   "main": ".eslintrc.js",
   "dependencies": {
-    "eslint-config-hackreactor": "^1.0.0"
+    "axios": "^0.19.2",
+    "eslint-config-hackreactor": "^1.0.0",
+    "file-system": "^2.2.2",
+    "mongoose": "^5.9.4",
+    "path": "^0.12.7"
```
Comment (jscholar) [inline package.json:11]: It seems like some of these dependencies are not needed (yet). I'm just reiterating what Josh said, which was to install packages and such as you need them.
Better organization of package versions. Less bulk on the application, etc.
Path and file-system are built in the node aren't they? Is there a reason to grab additional packages for them?

**Review by jscholar — APPROVED** [review]
Good stuff. Only thing I had to say was there were a couple of packages I think weren't necessary at this stage in the project.

File: package.json (hunk 2 above)
Comment (VinnieGlaser) [reply package.json:11]: I didn't know file-system and path were native to node, I'll look into removing those. I was using axios to grab my thousand pictures so its just lingering from that. I'll do my best to keep my dependencies low.

**Pull request merged**